blog/auth/auth.py

```diff
@@ -0,0 +1,43 @@
+from flask import Blueprint
+from flask_login import login_user, current_user, logout_user, login_required
+from blog.forms import ContactForm, RegistrationForm, LoginForm
+from flask import Flask, abort, flash, Markup, redirect, flash, render_template,request, Response, session, url_for
+from blog.models import User
+
+auth_blueprint = Blueprint('auth', __name__,
+                         template_folder='templates',
+                         static_folder='static')
+
+@auth_blueprint.route("/register", methods=['GET', 'POST'])
+def register():
+    if current_user.is_authenticated:
+        return redirect(url_for('/'))
+    form = RegistrationForm()
+    if form.validate_on_submit():
+        hashed_password = bcrypt.generate_password_hash(form.password.data).decode('utf-8')
+        user = User(username=form.username.data, email=form.email.data, password=hashed_password)
+        db.session.add(user)
+        db.session.commit()
+        flash('Your account has been created! You are now able to log in', 'success')
+        return redirect(url_for('logIn'))
+    return render_template('signup.html', title='Register', form=form)
+
+
+@auth_blueprint.route("/login", methods=['GET', 'POST'])
+def logIn():
+    if current_user.is_authenticated:
```

Comment:
I'm seeing this fragment a lot in your code.

I'd reccommend adding a method like 
```
def redirect_if_signed_in(current_user)
        return redirect(url_for('/'))
```

rather than having it repeated all the time